components/atom/input/src/Input/Component/index.scss

```diff
@@ -2,13 +2,15 @@ $bgc-atom-input-disabled: $c-gray-lightest !default;
 $c-atom-input-placeholder: $c-gray-dark !default;
 $m-atom-input-type-left: $m-m !default;
 $m-atom-input-type-right: $m-m !default;
+$bdrs-atom-input: 0 !default;
```

Comment:
please use `$bdrs-none` instead of 0 https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss#L4

Reply:
Done! 

What's the advantage of using `$bdrs-none`? Is there any scenario where `$bdrs-none` is going to be different from 0?

Reply:
I would say it's to have a common language with UX, we also have stuff like c-white